native/src/function_types.rs

```diff
@@ -89,3 +156,22 @@ pub fn get_network_from_chain_id(chain_id: &str) -> Network {
         _ => Network::Devnet,
     }
 }
+
+#[inline]
+pub fn parse_multisig_incomplete_session_bytes_argument(
```

Comment:
Following the naming convention, we can have

```suggestion
pub fn multisig_incomplete_session_bytes_argument(
```